src/core/include/openvino/op/gelu.hpp

```diff
@@ -13,9 +13,9 @@ namespace v0 {
 /// \brief Gaussian Error Linear Unit
 /// f(x) = 0.5 * x * (1 + erf( x / sqrt(2) )
 /// \ingroup ov_ops_cpp_api
-class OPENVINO_API Gelu : public Op {
+class OPENVINO_API Gelu : public util::UnaryElementwiseArithmetic {
```

Comment:
It is a sort of API change. Looks not safe since it becomes ABI incompatible

Reply:
OV doesn't claim ABI compatibility.
Discussed with @ilya-lavrenov and @ilyachur

Reply:
OpenVINO doesn't have requirements for ABI compatibility. It is a valid change. And no problem here.

Reply:
This change is logical, but changing header files in public API is not preferable.